README.md

```diff
@@ -297,6 +297,7 @@ Sometimes you have tests that work well for some platform combinations, e.g. Apa
 
 Tests can be altered using four lists:
 - `input` allows you to override global parameters in tests. An example usage is if you want to change the `dest_addr` of all tests to point to an external IP or host
+  - it includes also the `override_empty_host_header` flag: enabling it, empty Host headers will be replaced with `dest_addr`
```

Comment:
It feels like a breaking change. Are we OK with that? Shall we do it the other way around? Just loud thoughts.

Reply:
Being an override, part of the optional `testoverride` options, I felt that the correct default option would be to not override. Writing a test with an empty Host, I'm expecting that the Host is indeed sent empty.
If it sounds bad, I can turn it into `not_override_empty_host_header`

Reply:
Yes, it would be breaking, but I agree with @M4tteoP. I only know of one person who uses this feature and I don't think it would be too much to ask to make a small adjustment. We'll increase the major version number, that should suffice as a signal. @fzipi what do you think?

Reply:
This is part of the test yaml, right? Then we cannot change it without a new version of the yaml spec, sorry.

Reply:
So should I switch it to `not_override_empty_host_header `?
I think that #63  broke the default behavior, and enabling the custom behavior of overriding empty headers via `override_empty_host_header` would restore it.
We would have `dest_addr` that just overrides the destination address and the `override_empty_host_header` flag that force to override also the empty host headers, without any implicit behavior.

Reply:
I don't understand your answer @fzipi 😅. I vote for `override_empty_host_header` (i.e., the "breaking change").

Reply:
I don't mind the breaking change, as this is only in go-ftw. We should bail with an error for users that had the previous behavior, right?

Reply:
How would we do that? Wouldn't that still be a valid configuration (valid in the context of testing, not w.r.t. HTTP)?
- dest_addr override would be set
- Host header would be empty
- Host header override would be empty

If you wanted to test request spoofing, i.e., use a host that differs from the target address, then you can't raise an exception here, right?